app/src/test/java/tech/ula/utils/AssetFileClearerTest.kt

```diff
@@ -72,25 +78,27 @@ class AssetFileClearerTest {
     fun `Throws FileNotFoundException if files directory does not exist`() {
         filesDir.deleteRecursively()
 
-        assetFileClearer.clearAllSupportAssets()
+        runBlocking { assetFileClearer.clearAllSupportAssets() }
     }
 
     @Test
-    fun `Clears all assets and leaves filesystem structure intact`() {
+    fun `Clears all assets and leaves filesystem structure intact`() = runBlocking {
+        whenever(busyboxExecutor.recursivelyDelete(any())).thenReturn(SuccessfulExecution)
         assetFileClearer.clearAllSupportAssets()
 
-        assertTrue(filesDir.exists())
-        assertTrue(randomTopLevelFile.exists())
-        assertTrue(randomTopLevelDir.exists())
+        verify(busyboxExecutor, never()).recursivelyDelete(filesDir.absolutePath)
```

Comment:
That's a really neat feature of Mockito!  I'll keep this in mind when writing unit tests